app/views/content/train-to-be-a-teacher/teacher-degree-apprenticeships.md

```diff
@@ -47,20 +47,24 @@ The providers offering teacher degree apprenticeships are:
 * Xavier Teach Southeast, in partnership with the University of Sussex
 * University of Derby
 
-To view all courses for the TDA in secondary maths, please visit [Find Teacher Training](https://www.gov.uk/find-teacher-training-courses).
+[View all courses for the TDA in secondary maths](https://www.gov.uk/find-teacher-training-courses).
```

Comment:
Shouldn't we consider deeplinking here, given we've specified secondary maths in the link text, rather than make people go through the Find questions?

https://find-teacher-training-courses.service.gov.uk/results?age_group=secondary&applications_open=true&can_sponsor_visa=false&has_vacancies=true&l=2&subjects%5B%5D=G1&university_degree_status=false&visa_status=false

Note that this link only includes courses open to applications

Would need to check with policy team as to whether a) deeplinking is better (yes?) and if so b) parameters to be used such as whether to restrict to only open courses

Reply:
I'll check with policy, my worry was that it's not immediately obvious that it's a TDA (says it in the qualification bit but not at the top) so might confuse users. But it's a good shout so will double check.